assets_helper.py

```diff
@@ -28,7 +30,7 @@ def is_active(asset, at_time=None):
 
     if asset['is_enabled'] and asset['start_date'] and asset['end_date']:
         at = at_time or get_time()
-        return asset['start_date'] <= at <= asset['end_date']
+        return asset['start_date'] < at < asset['end_date']
```

Comment:
asset.start_date and asset.end_date has datetime/ timestamp datatype. 
Related to -> 
```
CREATE TABLE assets(asset_id text primary key, name text, uri text, md5 text, start_date timestamp, end_date timestamp, duration text, mimetype text, is_enabled integer default 0, nocache integer default 0, play_order integer default 0);
```
And then we will compare dates with 1 ms accuracy. I tested just now with one minute accuracy. It works correct. In my opinion this is not important, strict equality or not. But I used rule: upstream(pro version) is more correct than not upstream :D I can revert if it need...